os/arch/arm/src/amebad/amebad_enet.c

```diff
@@ -211,10 +215,16 @@ void up_netinitialize(void)
 		rtw_printf("Failed to register amebad netdev\n");
 	}
 #endif
+#ifndef CONFIG_VIRTUAL_BLE
 #if defined(CONFIG_AMEBAD_BLE) && defined(CONFIG_DRIVERS_BLE)
 	ameba_bm_dev_ble0 = bledev_register(&g_trble_drv_ops);
```

Comment:
why ops is opened here? it should be hided, below is an example of our style.
```
struct ble_lowhalf_s *priv, ble_lowhalf_s *vPriv;
int ret;
#ifdef CONFIG_AMEBAD_BLE
priv = amebad_ble_init();
#elif CONFIG_XXXX_BLE
priv = XXXX_ble_init();
#else
/* Nothing for now */
#endif
ret= bledev_register(priv);
#ifdef CONFIG_VIRTUAL_BLE
vPriv= virtual_ble_init();
ret = bledev_register(vPriv);
#endif
```

Reply:
g_trble_drv_ops is not changed in this PR. It is existing source code.